src/components/text.components.js

```diff
@@ -0,0 +1,31 @@
+import $ from 'jquery';
+import Component from './component';
+
+export default class TextInput extends Component {
```

Comment:
if the class name is TextInput, why the file name is text.component?

Reply:
adjusted to text component the class name and Email Component